services/madoc-ts/src/utility/deletion-utils.ts

```diff
@@ -0,0 +1,8 @@
+import fs from 'fs';
+
+export function removeIiifFromDisk(path: string) {
+  if (!fs.existsSync(path)) {
```

Comment:
I think we should add an additional check here to make sure the `path` starts with where we know the IIIF should be. Although unlikely, we don't want to delete things outside of that folder. 

The path can be found in a function: `saveManifestToDisk` in `services/madoc-ts/src/gateway/tasks/task-helpers.ts`

The path is:
```
${fileDirectory}/original/madoc-manifests/
```